cmd/notification.go

```diff
@@ -46,7 +47,13 @@ func (sys *NotificationSys) GetARNList() []string {
 	arns := []string{}
 	region := globalServerConfig.GetRegion()
 	for _, targetID := range sys.targetList.List() {
-		arns = append(arns, targetID.ToARN(region).String())
+		// httpclient target is part of ListenBucketNotification
+		// which doesn't need to be listed as part of the ARN list
+		// This list is only meant for external targets, filter
+		// this out pro-actively.
```

Comment:
Just commenting that `Exclude httpclient listeners` is good enough.

Reply:
Will keep the comment to indicate why this filtering is done..